sync-gateway-config.json

```diff
@@ -97,10 +96,12 @@ function(doc, oldDoc){
     access(doc.owner, "task-list." + doc._id + ".users");
   } else if (getType() == "task") {
     /* Write Access */
+    var owner = doc._deleted ? oldDoc.taskList.owner : doc.taskList.owner;
+    var list = doc._deleted ? oldDoc.taskList.id : doc.taskList.id;
     try {
-      requireAccess("task-list." + doc.taskList.id);
+      requireAccess("task-list." + list);
     } catch (e) {
-      requireUser(doc.taskList.owner);
+      requireUser(owner);
```

Comment:
Style comment only, but I think 'list_id' or 'listId' is a better name than just list for the temporary property.